runtime/Python2/src/antlr4/atn/ATNConfig.py

```diff
@@ -140,7 +140,7 @@ def __eq__(self, other):
             return False
         if self.passedThroughNonGreedyDecision != other.passedThroughNonGreedyDecision:
             return False
-        if self.lexerActionExecutor is not other.lexerActionExecutor:
+        if not(self.lexerActionExecutor == other.lexerActionExecutor):
```

Comment:
:question: Does `!=` not work in this case?

:memo: I'm not a Python programmer.